tests/framework/Models/External/tests

```diff
@@ -4,6 +4,11 @@
   input = 'all_methods.xml'
   csv = 'AllMethods/samples_out.csv'
  [../]
+ [./serialize_ext_model]
+  type = 'RavenFramework'
+  input = 'serialize_ext_model_and_use.xml'
+  csv = 'SerializeWrkd/samples_out.csv'
```

Comment:
if we include the pickle files as `out` results, then they'll be erased before the test starts right? That seems like it would be nice for the test.